tests/test_pane.py

```diff
@@ -69,7 +69,7 @@ def test_capture_pane(session: Session) -> None:
     session.new_window(
         attach=True,
         window_name="capture_pane",
-        window_shell='env -i PS1="$ " /usr/bin/env bash --norc --noprofile',
+        window_shell="/usr/bin/env PS1='$ ' bash --norc --noprofile",
```

Comment:
Would [`shutil.which`](https://docs.python.org/3/library/shutil.html#shutil.which) be worth doing, Extra portability?

Reply:
TBH I'm not sure how this would affect portability. I would highly assume that `bash` is always on `PATH` for all practical purposes.

Reply:
I've never came across a system which didn't have `env` at `/usr/bin/env` (as this is used in quite a lot of shebangs at the moment and this would break then). I've added some lookup anyhow (will rebase if accepted).

Reply:
thank you.

Reply:
you're right in all your points

i wouldn't hold anything past unix-like container/cloud and raspberry pi /single board-distros breaking convention.

Reply:
I've pushed a new version with the fixup commit fixed up :)